diff --git a/fitbit_web_api/configuration.py b/fitbit_web_api/configuration.py
index 1c5541d..5655672 100644
--- a/fitbit_web_api/configuration.py
+++ b/fitbit_web_api/configuration.py
@@ -16,6 +16,7 @@ import sys
 from logging import FileHandler
 from typing import Any, ClassVar, Dict, List, Literal, Optional, TypedDict, Union
 
+import aiohttp
 import urllib3
 from typing_extensions import NotRequired, Self
 
@@ -183,6 +184,7 @@ class Configuration:
         ssl_ca_cert: Optional[str] = None,
         retries: Optional[int] = None,
         ca_cert_data: Optional[Union[str, bytes]] = None,
+        pool_manager: aiohttp.ClientSession = None,
         *,
         debug: Optional[bool] = None,
     ) -> None:
@@ -278,6 +280,9 @@ class Configuration:
         """SSL/TLS Server Name Indication (SNI)
            Set this to the SNI value expected by the server.
         """
+        self.pool_manager: aiohttp.ClientSession = pool_manager
+        """Custom aiohttp PoolManager.
+        """
 
         self.connection_pool_maxsize = 100
         """This value is passed to the aiohttp to limit simultaneous connections.
diff --git a/fitbit_web_api/rest.py b/fitbit_web_api/rest.py
index 64f05ed..2cd3236 100644
--- a/fitbit_web_api/rest.py
+++ b/fitbit_web_api/rest.py
@@ -69,7 +69,7 @@ class RESTClientObject:
 
         self.retries = configuration.retries
 
-        self.pool_manager: Optional[aiohttp.ClientSession] = None
+        self.pool_manager: Optional[aiohttp.ClientSession] = configuration.pool_manager
         self.retry_client: Optional[aiohttp_retry.RetryClient] = None
 
     async def close(self) -> None:
